src/fold/models/baseline.py

```diff
@@ -49,12 +49,12 @@ def predict(
         self, X: pd.DataFrame, past_y: pd.Series
     ) -> Union[pd.Series, pd.DataFrame]:
         # it's an online transformation, so len(X) will be always 1,
-        return pd.Series(past_y.iloc[-1].squeeze(), index=X.index[-1:None])
+        return pd.Series(past_y.iloc[-1].squeeze(), index=X.index[-1:None]).fillna(0.0)
```

Comment:
now we need to fill up the initial NaNs for all transformations that may emit those